include/numerics/numeric_vector.h

```diff
@@ -191,36 +197,36 @@ class NumericVector : public ReferenceCountedObject<NumericVector<T>>,
   virtual std::unique_ptr<NumericVector<T>> clone () const = 0;
 
   /**
-   * Change the dimension of the vector to \p N. The reserved memory
-   * for this vector remains unchanged if possible.  If \p N==0, all
+   * Change the dimension of the vector to \p n. The reserved memory
+   * for this vector remains unchanged if possible.  If \p n==0, all
    * memory is freed. Therefore, if you want to resize the vector and
    * release the memory not needed, you have to first call \p init(0)
-   * and then \p init(N). This behaviour is analogous to that of the
+   * and then \p init(n). This behaviour is analogous to that of the
    * STL containers.
    *
    * On \p fast==false, the vector is filled by zeros.
    */
-  virtual void init (const numeric_index_type,
-                     const numeric_index_type,
-                     const bool = false,
-                     const ParallelType = AUTOMATIC) = 0;
+  virtual void init (const numeric_index_type n,
+                     const numeric_index_type n_local,
+                     const bool fast = false,
+                     const ParallelType ptype = AUTOMATIC) = 0;
```

Comment:
Hmm.. there must be a reason these variable names were removed in the first place.  Is there some compiler that's overzealous about its unused variable warnings?

Even if so, though, we should still keep the variable names, just wrapped in ```/* */```